stormpiper/stormpiper/spa/src/components/bmpForm.tsx

```diff
@@ -226,11 +240,16 @@ export const BMPForm = forwardRef(function BMPForm(props: formProps, ref) {
     let defaultValues: {
       [x: string]: string | number | boolean | null | undefined;
     } = {};
-    simpleStatus
-      ? (fieldSet = props.simpleFields)
-      : (fieldSet = props.allFields);
+    // simpleStatus
```

Comment:
can we remove new inactive code? new comments are welcome, but new inactive code is confusing.

Reply:
done